brainframe/cli/main.py

```diff
@@ -39,9 +39,12 @@ def main():
     # Exit with a clean error when interrupted
     def on_sigint(_sig, _frame):
         print()
-        if os_utils.current_command is not None:
+        if os_utils.current_command is None:
+            print_utils.fail_translate("portal.interrupted")
+        else:
+            # Let os_utils.run take care of bringing the process down when the current
+            # command is finished
             os_utils.current_command.send_signal(_sig)
```

Comment:
When would `os_utils.current_command` be `None`?

Reply:
That would be when there's no subprocess command running at that moment.

Reply:
So if there's a current subprocess, we kill it, but then don't run 
```python
print_utils.fail_translate("portal.interrupted")
```
?

Reply:
True. That's unideal. Let me see what I can do.

Reply:
Okay, I fixed this and added more guarantees.